src/Microsoft.DotNet.Darc/DarcLib/Models/GithubResourceConverters.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.DarcLib.Models;
+using Octokit;
+
```

Comment:
we should enable nullability in all of the new files.
Also, this one is not a model, I'd say it should go to the helpers?